### DISCUSSION TITLE
How we can switch network in wagmi 1.3.10?

### DISCUSSION THREAD

**Discussion by navdeep1998** [wevm/wagmi#2921] [Q&A]
Hi 
Can Anybody guide me how I can switch network in wagmi v1.3.10.According to the docs I've already tried But getting error.
![Screenshot from 2023-08-21 11-43-09](https://github.com/wagmi-dev/wagmi/assets/66154486/eb7b2bc4-3893-492f-903f-cb2403f6fbe4)
Wagmi : ^1.3.10

Please help to do this.Thanks

**Comment by Ruithlzz09** [comment]
Wrap you main application in below component and define configuration for wagmi 
`<WagmiConfig config={config}>{children}</WagmiConfig>`

Example of config
```
import {
  createConfig,
  configureChains,
  mainnet,
} from '@wagmi/core'
import { publicProvider } from '@wagmi/core/providers/public'
 
const { chains, publicClient, webSocketPublicClient } = configureChains(
  [mainnet],
  [publicProvider()],
)
 
const config = createConfig({
  autoConnect: true,
  publicClient,
  webSocketPublicClient,
})


```
